src/Migration/AutogenerateBookingForm.php

```diff
@@ -83,22 +90,31 @@ public function run(): MigrationResult
      */
     private function autoGenerateBookingForm(): void
     {
+        // Initialize the contao framework
+        $this->framework->initialize();
+
```

Comment:
Why do you Initiative the Contao framework herein?

Reply:
In line #L121 you have to call the method to load a data container array. As far as I know, the new migrations run independently from the Contao system, so you have to initialize the system first.

Reply:
Ah.... clearly. But then we should move this call down to line 119, shouldn't we?

Reply:
Yes, we can. ;) I've seen that you have already changed this order.